docu/docs/config.md

```diff
@@ -62,6 +62,54 @@ inputSource:
     mmPath: /opt/multimon/multimon-ng
 ```
 
+#### `lineIn:`
+|Feld|Beschreibung|Default|
+|----|------------|-------|
+|device|die device Id der Soundkarte|1|
+|mmPath|Pfad zur multimon-ng Binary|multimon-ng|
```

Comment:
@KirschbaumP mir ist gerade aufgefallen, dass hier vergessen wurde die Aufteilung in CARD und DEVICE nachzuziehen

Reply:
Einer von beiden ist aber in der Regel meist NULL oder? Denn könnte man per Default setzen und dadurch optional in der Konfig machen,

Reply:
Nein es müssen beide Werte gesetzt werden. Das `Card` ist die Soundkarte und `Device` beschreibt welchen Eingang auf der Soundkarte. Es gibt Soundkarten die einen getrennten Mikrofon und LineIn Eingang haben.

Reply:
Ah ok, dann sollte man wirklich beide Angeben, da hast du Recht!
Fehlen tut es in der Doku trotzdem :-P